src/NzbDrone.Core/MediaFiles/ScriptImportDecisionInfo.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections.Generic;
+using NzbDrone.Common.Disk;
+using NzbDrone.Core.Download;
+using NzbDrone.Core.Extras.Subtitles;
+using NzbDrone.Core.Parser.Model;
+
+namespace NzbDrone.Core.MediaFiles
+{
+    public class ScriptImportDecisionInfo
+    {
+        // needed
+        public bool isEpisodeFile { get; set; }
```

Comment:
This file should be going away, but properties should be CamelCase like they are in other instances.